cashweb-registry/src/store/topics.rs

```diff
@@ -113,6 +148,7 @@ impl<'a> DbTopics<'a> {
 
         let payload_hash = message.payload_hash().as_slice().to_vec();
 
+        // TODO: This should really use a merge operation that combines the burn_outputs.
```

Comment:
```rust
pub fn partial_merge(
    _key: &[u8],
    _existing_value: Option<&[u8]>,
    _operands: &mut rocksdb::MergeOperands,
) -> Option<Vec<u8>> {
    None // or find a way to merge `operands` somehow
}

fn full_merge(
    _key: &[u8],
    existing_value: Option<&[u8]>,
    operands: &mut rocksdb::MergeOperands,
) -> Option<Vec<u8>> {
    // apply operands (serialized `TopicPayload`s) onto existing_value (one `TopicPayload`)
}
```

where operands is an iterable over `&[u8]`, presumably the serialized TopicPayload

Reply:
How do these get wired up?